src/subdomains/voting/api/voting.controller.ts

```diff
@@ -55,6 +55,12 @@ export class VotingController {
     return this.votingService.getCurrentVotes();
   }
 
+  @Get('masternode')
+  @ApiOkResponse({ type: Number })
+  async getMasternodeCount(): Promise<number> {
+    return this.votingService.getMasternodeCount();
+  }
```

Comment:
Why not directly put it in masternode controller? With GET /masternode/voters or similar?